README.md

```diff
@@ -0,0 +1,42 @@
+# How it works
+
+## Scope
+
+The aim for this programme in CITANZ is to open a channel that connects our members to the enterprises for more opportunities. The job referral platform or, the workflow we provided, is based on a completely fair and neutral basis. We do not provide it as a service that a result is guaranteed.
+
+## Tools
+
+The job posting and application will be done through email, and the notification and communication will be done using Slack and/or Wechat, in our dedicated job referral channel, e.g. #jobs or the Wechat member group.
```

Comment:
Do we nominate a CITA working group member to be the coordinator for the communication? So that if someone has a JD that he or she can send it to the coordinator and we can format it and post it on the repo `issues`. I was wondering that how we address this part of comms in the flowchart. I can volunteer myself if finding a coordinator is a major concern.

Reply:
Sure. Thanks for the volunteering. As we discussed, @quintuschen  you will be responsible for the job referral program.